src/Magento/FunctionalTestingFramework/Console/TroubleShootCommand.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types = 1);
+
+namespace Magento\FunctionalTestingFramework\Console;
+
+use Codeception\Configuration;
+use Symfony\Component\EventDispatcher\EventDispatcher;
+use Codeception\SuiteManager;
+use Magento\FunctionalTestingFramework\Config\MftfApplicationConfig;
+use Magento\FunctionalTestingFramework\Exceptions\TestFrameworkException;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Magento\FunctionalTestingFramework\Util\ModuleResolver;
+
+class TroubleShootCommand extends Command
+{
+    const CODECEPTION_AUTOLOAD_FILE = PROJECT_ROOT . '/vendor/codeception/codeception/autoload.php';
+    const MFTF_CODECEPTION_CONFIG_FILE = ENV_FILE_PATH . 'codeception.yml';
+    const SUITE = 'functional';
+    const REQUIRED_PHP_EXTS = ['CURL', 'mbstring', 'bcmath', 'zip', 'dom', 'gd', 'intl'];
```

Comment:
I suggest to add all of this extensions to `composer.json`

Reply:
As discussed with team, I will remove php extension check from this command